libraries/apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo3/network/websocket/ServerMessage.kt

```diff
@@ -23,10 +23,9 @@ class ConnectionErrorServerMessage(val payload: ApolloJsonElement) : ServerMessa
  * A GraphQL response was received
  *
  * @param response, a GraphQL response, possibly containing errors.
- * @param complete, whether this is a terminal message for the given operation.
  */
 @ApolloExperimental
-class ResponseServerMessage(val id: String, val response: ApolloJsonElement, val complete: Boolean) : ServerMessage
+class ResponseServerMessage(val id: String, val response: ApolloJsonElement) : ServerMessage
```

Comment:
`ResponseServerMessage` is never terminal now
`OperationErrorServerMessage` is always terminal now